src/pipecat/services/elevenlabs.py

```diff
@@ -134,8 +136,9 @@ async def set_voice(self, voice: str):
         await self._connect()
 
     async def start(self, frame: StartFrame):
-        await super().start(frame)
-        await self._connect()
+        if not (self._websocket and self._websocket.open):
+            await super().start(frame)
+            await self._connect()
```

Comment:
I don't think we need this anymore. `StartFrame` is now guaranteed to arrive first to all frames. So the `self._connect()` should happen before any other frame is received.